python/nano/src/bigdl/nano/deps/openvino/pytorch/pytorch_openvino_model.py

```diff
@@ -35,8 +35,8 @@ def __init__(self, model, input_sample=None):
         if isinstance(model, torch.nn.Module):
             export(model, input_sample, 'tmp.xml')
             ov_model_path = 'tmp.xml'
-        AcceleratedLightningModule.__init__(self, None)
         OpenVINOModel.__init__(self, ov_model_path)
+        AcceleratedLightningModule.__init__(self, None)
```

Comment:
Why are we changing this order? Is this because `AcceleratedLightningModule` is the second inherited class?

Reply:
It doesn't matter to the function.. Just to follow the inheritance order..